### DISCUSSION TITLE
Causes of Framerate Issues

### DISCUSSION THREAD

**Discussion by orumetsu** [replit/kaboom#215] [Q&A] [answered]
So I recently made a game using kaboom.js, and there seems to be framerate issues on some laptop and PCs. Does the type of browser / browser version affect the performance of this, or is it just a low-end hardware issue?

**Comment by slmjkdbtl** [comment, marked as answer]
Hi! Do you mind sharing the code? The browser version shouldn't really matter. Also is it slow from the start or it builds up?

**Reply by orumetsu** [reply]
found the problem, turns out the display drivers wasn't installed yet
